apps/app/src/components/Comments.tsx

```diff
@@ -69,6 +70,13 @@ export const Comments = (props: CommentsProps): JSX.Element => {
     return <></>;
   }
 
+  const onCommentButtonClickHandler = () => {
+    mutate();
+    if (onCommentCountUpdated != null) {
+      onCommentCountUpdated();
+    }
```

Comment:
This one as well.

> I think avoiding null should be done at a higher level component.

> 

> If you need this onCommnetCountUpdated() process, please make sure that you do not need to pass mutate function as props on to other places that use PageComment component.

Reply:
I have removed `onCommentCountUpdated` props mutateCurrentPage from PageView